analyzer/src/functTest/resources/simple-existing-manipulation/build.gradle

```diff
@@ -12,8 +12,8 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.8.1'
-    compile group: 'org.hibernate', name: 'hibernate-core', version: '5.3.7.Final'
-    compile group: 'io.undertow', name: 'undertow-core', version: '2.0.15.Final'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.8.1'
+    implementation group: 'org.hibernate', name: 'hibernate-core', version: '5.3.7.Final'
+    implementation group: 'io.undertow', name: 'undertow-core', version: '2.0.15.Final'
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
```

Comment:
This is because compile/testCompile were removed from 7 ; is that correct?

Reply:
Yes, so at the very least, every test that is compatible with Gradle 7 needs to be changed.

I also tried to change as many build files as I could although sometimes "read-only" configuration is now an issue (I put a note). In order words, you could directly modify the `compile` configuration, but not the new ones.

Reply:
Thanks.